lib/elixir/lib/module.ex

```diff
@@ -725,7 +738,10 @@ defmodule Module do
       end
 
   """
-  def defines?(module, tuple) when is_tuple(tuple) do
+  @spec defines?(module, function_arity) :: boolean
+  def defines?(module, {function_macro_name, arity} = tuple)
```

Comment:
`function_macro_name` seems quite confusing to me, can we just say `function_name`?